lib/conversation/reply-dispatcher.js

```diff
@@ -1,12 +1,9 @@
 'use strict';
 
-const logger = require('winston');
-
-// TODO: Should this be a singleton?
+// TODO: Remove me?
```

Comment:
Sure, if we removed 80% of the abstraction, it makes sense to move the remainder.